execution_engine_testing/test_support/src/lib.rs

```diff
@@ -65,6 +65,12 @@ pub const DEFAULT_CHAIN_NAME: &str = "casper-execution-engine-testing";
 pub const DEFAULT_GENESIS_TIMESTAMP_MILLIS: u64 = 0;
 /// Default maximum number of associated keys.
 pub const DEFAULT_MAX_ASSOCIATED_KEYS: u32 = 100;
+/// Default max serialized size of `StoredValue`s.
+#[deprecated(
+    since = "2.2.0",
```

Comment:
Should probably be since 2.3.0 as that's going to be the next published version.  Same comment applies to all deprecation notices added to this crate in this PR.

Reply:
Done.